airflow/ui/src/components/FlexibleForm/FieldNumber.tsx

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 import type { FlexibleFormElementProps } from ".";
-import { paramPlaceholder, useParamStore } from "../TriggerDag/useParamStore";
+import { paramPlaceholder, useParamStore } from "../../queries/useParamStore";
```

Comment:
```suggestion
import { paramPlaceholder, useParamStore } from "src/queries/useParamStore";
```

Let's replace all of these nested relative paths